### ISSUE TITLE
Accessibility bug: touch target of all the post options is too small

### ISSUE THREAD

**Issue by SleeplessByte** [bluesky-social/social-app#6312]
### Steps to Reproduce

1. Go to https://bsky.app/
2. Inspect any of the post option buttons' touch target
3. Realise that a thick thumb wil have difficulty pressing it

In replies the size is 28x28, and in highlighted / opened posts it's 32x32. 

- [ ] Ensure the touch target of each button is at least 44x44

For more information: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

### What platform(s) does this occur on?

Web (Desktop), Web (Mobile)

### Device Info

Not relevant

### What version of the app are you using?

Web

### Additional Information

You don't need to physically increase the size of the visible icon. Adding padding + negative margin is an effective method to increase the touch target without moving the element around.

**Comment by stubbornella** [comment]
WCAG 2.2 requires one of two options:

1. Size your clickable targets at least 24px by 24px.
2. If your design requires a smaller clickable target, keep any other clickable elements at least 12px from the center of the target.

This is sufficient because browsers _kind of_ expand the tap target of elements and try to find the element the user intended to click. It works slightly differently in each browser, but the end result is roughly the same: Browsers assist users who are unable to click perfectly given the combination of their circumstances and abilities.

It’s a bit more complex than that. WCAG is a good starting point, or I wrote a post: https://www.stubbornella.org/2023/09/17/expanding-your-touch-targets/

**Comment by SleeplessByte** [comment]
Thank you @stubbornella for that response. Whilst the 2.2 guidelines have past my desk, I have not yet memorised them by heart and you are 100% correct, of course. I think the information in your post also provides very valuable context about the fact that "hard to touch" elements are made easier to touch by various user agents. 

As I understood it, however, is that the app design guidelines still expect [44px by 44px](https://developer.apple.com/design/human-interface-guidelines/buttons#Best-practices) or 48dp by 48dp, because unlike browsers, those mobile OSes don't always expand the tap target (but are getting better at it), but then again [not everyone agrees](https://medium.com/@shoobe01/44-px-why-apple-is-wrong-6bd6f6846871). 

I think the point of those who disagree matches exactly your second item from [WCAG 2.2](https://www.w3.org/TR/WCAG22/#target-size-minimum) which is more or less make sure mis-clicking doesn't affect a user badly. The specific text @stubbornella is referring to is:

> Spacing: Undersized targets (those less than 24 by 24 CSS pixels) are positioned so that if a 24 CSS pixel diameter circle is centered on the [bounding box](https://www.w3.org/TR/WCAG22/#dfn-bounding-boxes) of each, the circles do not intersect another target or the circle for another undersized target;

WCAG 2.2 moved the [44px by 44px to level AAA](https://www.w3.org/TR/WCAG22/#target-size-enhanced).

------------------

That said, even with 24px by 24px, bsky.app is still not in compliance because misclicking on a reply post will *open that post*, which is extremely jarring, and _moves the actions from under your finger_. In terms of the guidelines, I think we can agree that that means that the various interactable elements are not, in fact, 12px apart. 

What do you think?

**Comment by stubbornella** [comment]
The WCAG 2.2 guidelines are new-ish so we are all figuring out the best way to meet the criteria. I think of the new guidelines as a partnership between the browser and the developer to each do their part to make a great user experience. It allows more flexibility for the designer by having the browser be smarter about interpreting clicks.

Devs need to meet either of the criteria, not both. With a 24px by 24px icon, they meet the first criteria. If you’d like to verify if it meets the second criteria, you can measure 12px from the center of the icon in any direction and test if the click activates any other events.

Are you testing the feed view or the post view?

**Comment by SleeplessByte** [comment]
The biggest issue for the person who told me they had an issue with this is the reply post in the feed few, where the icons are a circle of diameter 28px, and a missclick opens the post in +full view+, but that's basically #6305

So I suppose it's the feed view, but specifically replies to posts and not posts themselves.
